### PR TITLE
MMM load updates

### PR DESCRIPTION
**Pull request by michaelraczycki**
fixed dims format, added save_load test to delayed_saturated_mmm. Now MMM classes can properly use save/load functionality

----
:books: Documentation preview :books:: https://pymc-marketing--317.org.readthedocs.build/en/317/

### CODE REVIEW

File: tests/mmm/test_delayed_saturated_mmm.py [hunk 1]
```diff
@@ -293,3 +293,17 @@ def test_data_setter(self, toy_X, toy_y):
             )
         except Exception as e:
             pytest.fail(f"_data_setter failed with error {e}")
+
+    def test_save_load(self, mmm_fitted):
+        model = mmm_fitted
+
+        model.save("test_save_load")
```
Comment (juanitorduz) [inline tests/mmm/test_delayed_saturated_mmm.py:300]: shall we delete the file after the test ends? related to https://github.com/pymc-labs/pymc-marketing/issues/316

Comment (michaelraczycki) [reply tests/mmm/test_delayed_saturated_mmm.py:301]: I've added a line removing the temp file after the test is finished

**Comment by michaelraczycki** [comment]
the failure of test CI is expected, as a release on pymc-experimental is needed. Please add your thoughts regardless, so we can proceed when release is done

File: tests/mmm/test_delayed_saturated_mmm.py (hunk 1 above)
Comment (juanitorduz) [reply tests/mmm/test_delayed_saturated_mmm.py:301]: great! maybe we can remove the file of https://github.com/pymc-labs/pymc-marketing/issues/316 here :) ?

Comment (michaelraczycki) [reply tests/mmm/test_delayed_saturated_mmm.py:301]: I'm not sure what file we're talking about, I think the issue mentions something that was created and left on local of the person running the tests, not something I can do. But they shouldn't be left over after running tests anymore after that

Comment (juanitorduz) [reply tests/mmm/test_delayed_saturated_mmm.py:301]: ah ok! thanks!

**Comment by twiecki** [comment]
@ricardoV94 can we merge this?

**Pull request merged**